File: MPI/mpicheck/tests/functions/1.0/MPI_Op_create.f90

        
subroutine var_0(val, ierr)
    integer val, ierr
end subroutine
        program main
        use mpi
        
        EXTERNAL var_0
       LOGICAL var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_op_create(var_0, var_1, var_2, var_3)
       call pmpi_op_create(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Op_create.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_User_function *var_0;
    int var_1;
    MPI_Op *var_2;
    int ret;
    /* calls */
    ret = MPI_Op_create(var_0, var_1, var_2);
    ret = PMPI_Op_create(var_0, var_1, var_2);
    return 0;
}
